webcompat/static/js/lib/wizard/ui-utils.js

```diff
@@ -52,8 +69,19 @@ export const showContainer = (container, cb) => {
   });
 };
 
+const hideContainerReduced = (container) => {
+  const domElement = container[0];
+  domElement.style.display = "none";
+  container.removeClass("open");
+};
+
 export const hideContainer = (container) => {
   const domElement = container[0];
+
+  if (isReducedMotion()) {
+    return hideContainerReduced(container);
+  }
```

Comment:
Is there a reason to have a second hide-method instead of setting the `duration` in the animation-call below to 0? Intuitively, it feels like it would be simpler to do that, but my intuition might be wrong.

Reply:
I've tried it yeah, but the function inside `anime.begin` callback doesn't run with duration: 0 :(
```js
    begin: () => {
      domElement.style.display = "flex";
    },
```
(this is basically what's making next step visible).

Tried to increase duration to 20 or 40, then it works most of the time and sometime doesn't, which seemed pretty fragile, so I just decided to have a separate function to have more control.

Reply:
Ah, that's a shame. Thanks for explaining!